TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/feature_selection/MenuFeatureSelectionController.java

```diff
@@ -0,0 +1,70 @@
+package com.novoda.tpbot.feature_selection;
+
+import android.content.Context;
+import android.support.annotation.MenuRes;
+import android.util.SparseArray;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.MenuItem;
+
+import com.novoda.tpbot.R;
+
+public final class MenuFeatureSelectionController implements FeatureSelectionController<Menu, MenuItem> {
```

Comment:
Handles the inflation of the `Menu` and the individual `MenuItem`s. This class also handles the toggle of menu items.

Reply:
Want to try and remove as much code from the `activity` as possible.